aws-java-sdk-core/src/main/java/com/amazonaws/retry/PredefinedRetryPolicies.java

```diff
@@ -217,7 +221,7 @@ public final long delayBeforeNextRetry(AmazonWebServiceRequest originalRequest,
                 scaleFactor = SCALE_FACTOR;
             }
 
-            long delay = (1 << retriesAttempted) * scaleFactor;
```

Comment:
Hi @amrasarfeiniel, may I withdraw my original comment/vote.  Just realized I misread the code.  I know this is minor, but 1L is nicer and more correct, and reflect the intent (of using the long type) more closely.  My apologies.